### PR TITLE
Relax requests constain

### PR DESCRIPTION
**Pull request by marcelotrevisani**

### CODE REVIEW

**Review by chr1sdi — APPROVED** [review]
LGTM definitely needs relaxing.

**Review by chr1sdi — DISMISSED** [review]
LGTM

File: setup.py [hunk 1]
```diff
@@ -21,6 +21,6 @@
     ],
     python_requires='>=3.7',
     install_requires=[
-        'requests==2.24.0'
+        'requests>=2.24.0'
```
Comment (yogeshrnaik) [inline setup.py:24]: @marcelotrevisani 
Please also change requests version in `setup.cfg`
https://github.com/camunda-community-hub/camunda-external-task-client-python3/blob/ed7a493942dea6c1b570298f67b09dc12ff7da11/setup.cfg#L26

**Review by yogeshrnaik — APPROVED** [review]

**Comment by yogeshrnaik** [comment]
Closing this PR as it is no longer relevant. requests version is already relaxed in master branch

**Pull request closed without merging**